frontend/src/metabase/parameters/components/ParameterValuePicker/ListPicker/ListPicker.unit.spec.tsx

```diff
@@ -21,6 +21,10 @@ const VALUES = [
   "1-799 Smith Road",
 ];
 
+const userEvent2 = userEvent.setup({
+  advanceTimers: jest.advanceTimersByTime,
+});
```

Comment:
if we use this userEvent in tests that don't use fake timers, testing library gets kinda cranky